legacy-registries-v0/src/main/java/io/github/minecraftcursedlegacy/api/registry/TileEntities.java

```diff
@@ -0,0 +1,47 @@
+package io.github.minecraftcursedlegacy.api.registry;
+
+import io.github.minecraftcursedlegacy.accessor.registry.AccessorTileEntity;
+import net.minecraft.tile.entity.TileEntity;
+
+import java.util.Locale;
+
+/**
+ * Utilities for registering and retrieving tile entity classes.
+ */
+public final class TileEntities {
+	/**
+	 * Adds a registered tile entity.
+	 * @param clazz the class of the tile entity to register.
+	 * @param id the ID to register the tile entity as.
+	 */
+	public static void registerTileEntity(Class<? extends TileEntity> clazz, Id id) {
```

Comment:
now that it doesn't actively rely on the registries module, I think making a separate module for tile entities would be best.